icefall/score_estimator.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import glob
+import logging
+from pathlib import Path
+from typing import Tuple, List
+
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+from torch.utils.tensorboard import SummaryWriter
```

Comment:
Please remove unused imports.
If you have a look at https://icefall.readthedocs.io/en/latest/contributing/code-style.html
and start to use `flake8`, it will tell you this import is never used.